js/client-teambuilder.js

```diff
@@ -2201,7 +2201,7 @@
 
 				var changed = !set.ivs || set.ivs[stat] !== val;
 				if (changed) {
```

Comment:
Hm, this might make it too easy to accidentally turn off auto IVs? Maybe the IV choosers should be grayed out if auto-IVs is checked.